src/labb4_ETSN20/SearchInFile.java

```diff
@@ -10,23 +10,24 @@ public static void main(String args[]) throws FileNotFoundException{
 		
 		SearchInFile search = new SearchInFile();
 		search.searchLines(args[0], args[1]);
```

Comment:
String parameter 1 = args[0];
String parameter 2 = args[1];
A bit confusing with the parameters